README.md

```diff
@@ -2,61 +2,129 @@
+4. Create a `Firebase Realtime Database` and update its ruleset to allow read and write access.
+5. Update the Firebase configuration object in `config/config.js` with app-specific config details obtained from the Firebase console app settings.
 
-    > Ex : <meta-data
+##### Google Maps Setup:
 
-        android:name="com.google.android.geo.API_KEY"
-        android:value=**"AIzaSyDmwJddIPTcALyZtj7p9mFFlkMvpMkati8"**/>
+1. Generate a `Google Maps API Key` using this [Link](https://mapsplatform.google.com/) for G-social's map view, or use the provided key (not guaranteed to always work). Ensure the key has all maps, routes, and places enabled
+2. After obtaining your Google Maps API key, replace the example key with your own.
+   > - For Android open `android/app/src/main/AndroidManifest.xml` and replace
 
-1.  Find the file name as `strings.xml` located in `android/app/src/main/res/values`. Place your **Facebook app id** in there.
+```sh
+<meta-data
+    android:name="com.google.android.geo.API_KEY"
+    android:value="AIzaSyDmwJddIPTcALyZtj7p9mFFlkMvpMkati8"
+/>
+```
 
-    > Ex: <string name="facebook_app_id">**2349388348405699**</string>
+> - For IOS open `ios/Go_social/AppDelegate.m` and replace
 
-So now you are ready to run Go-social.
+```sh
+[GMSServices provideAPIKey:@"AIzaSyDmwJddIPTcALyZtj7p9mFFlkMvpMkati8"];
+```
 
-## Run Go-social
+##### Facebook Authentication Setup:
 
-- Run the app
+1. To enable `Facebook login` for Go-social, obtain a Facebook app id using the provided [link](https://developers.facebook.com/) and enable both email/password and Facebook sign-in method in Firebase.
+2. After setting up Facebook authentication, use your Facebook App ID and App Secret to enable the "Facebook" `Sign-in method` in Firebase.
+3. Update the following file in your repository:
+   > - For Android open `android/app/src/main/res/values/strings.xml` and replace
 
-  `npx react-native run-android`
+```sh
+<string name=“facebook_app_id”>2349388348405699</string>
+```
 
-  Then enable the remote Js debugging.
+> - For iOS open `ios/Go_social/info.plist` and replace
+
+```sh
+...
+<string>fb{YOUR_APP_ID}</string>
+...
+<key>FacebookAppID</key>
+<string>{YOUR_APP_ID}</string>
+<key>FacebookDisplayName</key>
+<string>{YOUR_APP_NAME}</string>
+...
+```
+
+### Run Go-social
+
+Now you are ready to run Go-social
+(for android)
+
+```sh
+npx react-native run-android
+```
+
+(for iOS)
+
+```sh
+npx react-native run-ios
+```
+
+## Contributing
+
+Contributions to this project are welcome! To contribute, follow these steps:
+
+[Contribution Guidelines](https://github.com/scorelab/Go-social/blob/14e5dfcdfb59888efc318bb4835e3577d5d09532/.github/CONTRIBUTING.md)
+
+1. Fork the repository to your own GitHub account
+2. Clone the forked repository to your machine
+3. Make your changes and commit them with descriptive commit messages
+4. Push your changes to your forked repository
+5. Open a pull request from your forked repository to the original repository
```

Comment:
This should be in contribution guideline md file

Reply:
Updated contributing guideline